### PR TITLE
docs: Add "Better Together" colour

### PR DESCRIPTION
**Pull request by Binarytales**
screen shot 2019-02-27 at 2 31 45 pm

This also adds a little but of positive feedback to click on a color to get the variable name in your clipboard

### CODE REVIEW

File: packages/axiom-materials/src/colors.js [hunk 1]
```diff
@@ -171,6 +171,8 @@ export const socialInstagramActive = { r: 171, g: 58, b: 133 };
 export const socialInstagram = { r: 183, g: 71, b: 146 };
 export const socialInstagramHover = { r: 196, g: 86, b: 160 };
 
+export const betterTogther = { r: 240, g: 78, b: 78 };
```
Comment (siggysamson) [inline packages/axiom-materials/src/colors.js]: Looks like a small typo here - the `e` got lost! `betterTogether`

File: site/components/Documentation/Resources/Materials/Colors/ColorGrid.css [hunk 2]
```diff
@@ -8,6 +8,10 @@
   margin: 0 auto;
   border: 0.0625rem solid var(--color-theme-border);
   border-radius: 50%;
+  cursor: pointer;
+}
+.si-color-dot:active {
+ box-shadow: 0 0 0 3px var(--color-theme-border);
```
Comment (siggysamson) [inline site/components/Documentation/Resources/Materials/Colors/ColorGrid.css]: If I'm remembering this correctly we didn't use `px` much in Axiom and usually went with `rem`. While I don't mind it much it's probably worth considering that for the sake of consistency

File: packages/axiom-materials/src/colors.css [hunk 3]
```diff
@@ -198,5 +198,7 @@
   --color-social-instagram--active: rgb(171, 58, 133);
   --color-social-instagram: rgb(183, 71, 146);
   --color-social-instagram--hover: rgb(196, 86, 160);
+
+  --colour-better-together: rgb(240, 78, 78);
```
Comment (t-sauer) [inline packages/axiom-materials/src/colors.css]: Shouldn't this be `color` to stay consistent with the other names?

File: packages/axiom-materials/src/colors.js (hunk 1 above)
Comment (t-sauer) [inline packages/axiom-materials/src/colors.js]: Small typo here (you are missing an `e` in `Together`)

File: site/components/Documentation/Resources/Materials/Colors/ColorGrid.css (hunk 2 above)
Comment (t-sauer) [inline site/components/Documentation/Resources/Materials/Colors/ColorGrid.css]: Indentation seems to be off here.

Comment (roburidge) [inline site/components/Documentation/Resources/Materials/Colors/ColorGrid.css]: nice addition 👍

**Comment by Binarytales** [comment]
Thanks for comments @t-sauer - This is why I shouldn't code with a cold 😰

File: site/components/Documentation/Resources/Materials/Colors/ColorGrid.css (hunk 2 above)
Comment (Binarytales) [reply site/components/Documentation/Resources/Materials/Colors/ColorGrid.css]: Indeed. The linter picked this up to and I have switch to `rem`

**Comment by Binarytales** [comment]
@siggysamson I wasn't sure about the `docs` vs `feat` thing. Katja asked for this and it's a colour that should _not_ ever be used in real UIs.

**Comment by siggysamson** [comment]
Oohh interesting! Makes sense to me. In that case I'd leave it with `docs`! Thanks for clarifying

**Pull request merged**